core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2067,8 +2067,8 @@ class SparkContext(config: SparkConf) extends Logging {
   }
 
   /**
-   * Run a job on all partitions in an RDD and pass the results to a handler function, but take
-   * a function of type `Iterator[T] => U` instead of `(TaskContext, Iterator[T]) => U`.
+   * Run a job on all partitions in an RDD and pass the results to a handler function, this time the
```

Comment:
If I understood you correctly, both of the functions should have `X` as the description, but the one with `TaskContext` has this added to its description: `The function that is run against each partition should additionally take TaskContext argument.`, right? 
At the same time the actual argument description stays the same in both cases because the `TaskContext` thing is already described in the method signature.